modules/client-configuration/pages/vhm.adoc

```diff
@@ -0,0 +1,38 @@
+[[virt-vhm]]
+= Virtual Host Managers
+
+Virtual Host Managers (VHMs) are used to gather information from a range of client types.
+
+You can use a VHM to manage public cloud instances that are not directly supported by {productname}.
```

Comment:
IMHO it would be better to rephrase this to something like:

_You can use a VHM to collect your Private and Public Cloud instances and expose them into SUSE Manager_

I think it's not correct to say "manage public cloud instances that are not directly **supported** by SUSE Manager"

Reply:
Those sentences are saying different things, though:

The first sentence says "VHMs can manage VM clients".
The second sentence says "VHMs can manage VM clients that aren't managed in other ways".

I agree that use of the word "supported" is confusing, though. Basically, we have Xen and KVM that are managed directly, and we have VHMs which are used for managing everything else. How would you word that?

Reply:
The thing is that with VHM you cannot really manage the collected virtual instances. You only see them as virtualization instances.

Only in case that some of those virtual instances are actually registered as clients in SUMA (trad or minion), then you would be able to manage them.

In an ideal scenario, you would have all your virtual instances already registered in SUMA, then after adding the VHM that host those instances, you would get the proper virtualization groups and will see those instances now as belonging to this particular VHM. This is really important for a proper subscription matching (specially for 1-2 virtual machine subscription).

@Loquacity in case you want to clarify this better, we can maybe have a meeting in mumble. Just ping me :smile: :+1:

Reply:
OK, I've changed "managed" to "communicate" in this instance. I think a little more exposition might be required, though.